### PR TITLE
samples: fix klocwork fiunitialized infilename variable

### PR DESCRIPTION
**Pull request by generalova-kate**
### Details:
 - fix klocwork: 'this->filename' is not initialized in this constructor.

### CODE REVIEW

File: inference-engine/samples/common/utils/include/samples/csv_dumper.hpp [hunk 1]
```diff
@@ -18,7 +18,7 @@
  */
 class CsvDumper {
     std::ofstream file;
-    std::string filename;
+    std::string filename = "";
```
Comment (ilya-lavrenov) [inline inference-engine/samples/common/utils/include/samples/csv_dumper.hpp:21]: string is initialized by a default ctor. Why do we need to do it explicitly?

Comment (generalova-kate) [reply inference-engine/samples/common/utils/include/samples/csv_dumper.hpp:21]: to avoid the Klocwork issue or I can ignore it.

Comment (ilya-lavrenov) [reply inference-engine/samples/common/utils/include/samples/csv_dumper.hpp:21]: we need to mark that issue as false alarm

Comment (generalova-kate) [reply inference-engine/samples/common/utils/include/samples/csv_dumper.hpp:21]: ok, I closed PR

**Pull request closed without merging**